tests/unit/common/browserInfoHelper.test.js

```diff
@@ -250,11 +264,11 @@ describe("BrowserInfoHelper", () => {
   });
 
   it("getScreen", async () => {
-    jest.spyOn(globalsUtil, "getScreen").mockReturnValue({
+    screenSpy.mockImplementation(() => ({
       width: "900px",
       height: 1021,
-      colorDepth: 17,
-    });
+      colorDepth: 17
+    }))
```

Comment:
Minor: Please add the semi colon at the end to follow the same pattern everywhere.